accounts/views.py

```diff
@@ -91,8 +93,14 @@ def form_valid(self, form):
                 slack_user = lookup_user(self.object.email)
                 if slack_user:
                     user_add(settings.SLACK_TARGET_EXEC, slack_user)
-
-            # TODO: Add new active members to the active channel in Slack (and remove inactive / alumni etc.)
+            elif active_group not in oldgroups:
+                slack_user = lookup_user(self.object.email)
+                if slack_user:
+                    user_add(settings.SLACK_TARGET_ACTIVE, slack_user)
```

Comment:
Wouldn’t this add anyone who didn’t have active status before? Even if they still don’t?